src/interfaces/alteration-interfaces/alteration-interface.ts

```diff
@@ -26,9 +25,11 @@ export interface AlterationIF extends CoopAlterationIF {
     identifier: string
     legalType: CorpTypeCd
   }
-  /** This is different from bcrs-shared-components/interfaces/name-request-interface.ts
-   * and should be refactored in the future. */
-  nameRequest?: NameRequestIF
```

Comment:
This is an example of using the local interface with its extra properties (for data exchange with Legal API). I deleted that interface, so what this object looks like now is in the new code >>>

There's actually a type for it but it contains more properties that we don't care about, so I just did this.